src/osd/PrimaryLogPG.cc

```diff
@@ -6006,13 +6006,10 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 	epoch_t pg_epoch = info.history.same_interval_since;
 	tracepoint(osd, do_osd_op_pre_assert_interval, soid.oid.name.c_str(), soid.snap.val, epoch);
 	dout(5) << "Check interval " << soid << " epoch " << epoch << " pg epoch " << pg_epoch << dendl;
-	if (epoch > pg_epoch) {
+	if (epoch != pg_epoch) {
```

Comment:
this one is confusing to me - won't the epoch be different if there have been any osdmap changes that don't affect this pg? or is scrub reporting the same_interval_since epoch?